### PR TITLE
use env file for docker-compose

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1 +1,9 @@
 SECRET_API_TOKEN=somethingThatMustChange
+RECAPTCHA_SITE_KEY=6LfClyATAAAAAD19Z-ReiK4xsGVCMWuTF4f9FMr_
```
Comment (VoyTechnology) [inline .env.example]: Intentional?

Comment (butlerx) [inline .env.example]: site key is public

File: .env.example [hunk 2]
```diff
@@ -1 +1,9 @@
...
+RECAPTCHA_SECRET_KEY=
+LOG_ROTATE=24h
+SMTP_HOST=mailhost.redbrick.dcu.ie
+SMTP_PORT=25
+SMTP_SECURE=false
```
Comment (VoyTechnology) [inline .env.example]: This was true in the .yml file

Comment (butlerx) [inline .env.example]: yes but the port and mailhost were also different
eg it was all wrong

**Pull request merged**